sample/F8/tracks

```diff
@@ -0,0 +1,202 @@
+[
+   {
+      "id":1,
+      "name":"ADS & MONETIZATION",
+      "color":"#BF532E",
+      "description":null,
+      "location":"C",
```

Comment:
The fields should be removed. The current API docs are anyway deprecated due to the new API which is being built. 

We will no longer be depending on the API output for the the sample json format (since the API output will be following json api spec and is not suited for such import/export purposes). The sample will be have a separate format definition defined in the repo.